### PR TITLE
Hello-microverse

### PR DESCRIPTION
**Pull request by djo1975**
I added:
index.html
style.css
readme.md

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- [👥 Authors](#authors)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+Is a practice project . 
+
+
```
Comment (christianbiring1) [inline readme.md]: - [x] Kindly make sure your README has a detailed description telling what the project is about.
`Hint: You may consider taking` [this](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=Estimated%20time%3A%200.5h-,Description,-In%20this%20project) for reference.

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (christianbiring1) [inline Index.html]: - [x] Kindly make sure your CSS file is linked correctly with your HTML
`Hint: You may consider adding <link rel="stylesheet" href="styles.css">`

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @djo1975 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose


## Highlights
- [ ] The linters were set up correctly :+1: 
- [ ] The correct README was used :+1: 

## Required Changes ♻️
- [ ] `KINDLY MAKE SURE ALL YOUR LINTERS CHECKS ARE GREEN` :heavy_check_mark: 
Hint: You may consider running the command `npx stylelint "**/*.{css,scss}" --fix` to fix the style linters that are failing.
You may consider checking the entire documentation [here](https://github.com/microverseinc/linters-config/tree/master/html-css)

- [ ] `KINDLY MAKE SURE YOUR PR HAS A DESCRIPTIVE title telling what changes were introduced in the project.`
`Hint: Hello-microverse` would sound better than `MyFirstPull`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by djo1975** [comment]
hi
now I fixed
best regards Mladan

**Review by RiyaBulia12 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 
Hi @djo1975 👋🏽  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congratulations-Memes-FEATURED
## Highlights
* Correct README template ✅ 
* Linters setup correctly 🟢 
* Correct GitHub Flow 👍🏽 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**